src/libstd/ffi/c_str.rs

```diff
@@ -355,6 +355,41 @@ impl CString {
         CString { inner: v.into_boxed_slice() }
     }
 
+    /// Creates a C-compatible string by reading from an object that implements ```Read```.
+    /// It reads all characters including the null character.
+    ///
+    /// Because bytes are checked on null characters during the reading process,
+    /// no extra checks are required to ensure that no null characters precede the
+    /// terminating null character.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::ffi::CString;
+    ///
+    /// let test = "Example\0";
+    /// let string = CString::from_reader(test.as_bytes()).unwrap();
+    /// ```
+    ///
+    #[stable(feature = "cstring_from_reader", since = "1.33.0")]
```

Comment:
```suggestion
    #[unstable(feature = "cstring_from_reader", issue = "<new_issue_number>")]
```
Please replace `<new_issue_number>` with a fresh issue you've created.

Reply:
Thanks for your feedback! First time here :) I will go change it right away!

Reply:
Done (Y)!